### DISCUSSION TITLE
Hello everyone!

### DISCUSSION THREAD

**Discussion by VAI7899** [github/india#356] [Students]
For claiming students developer pack, do we need to complete the first module in the self-learning path and update it in LinkedIn?

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by TheSuranaverse** [comment]
@VAI7899 
All the queries & their solution is compiled here, refer to the link below. Also, click on 🌟 to bookmark it. If this answers your question, mark it as **answered**!
Link: https://github.com/Neklaustares-tPtwP/Microsoft-Future_Ready_Talent-Internship/tree/master/Hot%20Queries%20%26%20Solution

Thank you!
